VideoWeb/VideoWeb/ClientApp/angular.json

```diff
@@ -1,155 +1,176 @@
 {
-  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
-  "version": 1,
-  "newProjectRoot": "projects",
-  "projects": {
-    "VideoWeb": {
-      "root": "",
-      "sourceRoot": "src",
-      "projectType": "application",
-      "prefix": "app",
-      "schematics": {},
-      "architect": {
-        "build": {
-          "builder": "@angular-devkit/build-angular:browser",
-          "options": {
-            "progress": true,
-            "extractCss": true,
-            "outputPath": "dist",
-            "index": "src/index.html",
-            "main": "src/main.ts",
-            "polyfills": "src/polyfills.ts",
-            "tsConfig": "src/tsconfig.app.json",
-            "assets": [
-              "src/assets",
-              {
-                "glob": "*",
-                "input": "node_modules/govuk-frontend/govuk/assets/fonts",
-                "output": "assets/fonts/"
-              },
-              "src/scripts"
-            ],
-            "styles": ["src/styles.css", "src/styles.scss", "node_modules/ngx-toastr/toastr.css"],
-            "stylePreprocessorOptions": {
-              "includePaths": ["src/sass"]
-            },
-            "scripts": [
-              "src/scripts/pexrtc.js",
-              "src/scripts/EdgePF.js",
-              "src/scripts/IEPF.js",
-              "src/scripts/Heartbeat.js",
-              "node_modules/govuk-frontend/govuk/all.js",
-              "node_modules/chart.js/dist/Chart.js",
-              "node_modules/@mediapipe/camera_utils/camera_utils.js",
-              "node_modules/@mediapipe/control_utils/control_utils.js",
-              "node_modules/@mediapipe/drawing_utils/drawing_utils.js",
-              "node_modules/@mediapipe/selfie_segmentation/selfie_segmentation.js"
-            ],
-            "aot": true
-          },
-          "configurations": {
-            "production": {
-              "fileReplacements": [
-                {
-                  "replace": "src/environments/environment.ts",
-                  "with": "src/environments/environment.prod.ts"
+    "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
```

Comment:
are there any changes made in this json file or just format change ?

Reply:
There is a temporary unit test exclusion but the rest is formatting.